playbooks/common/openshift-master/config.yml

```diff
@@ -47,6 +47,7 @@
           public_hostname: "{{ openshift_public_hostname | default(None) }}"
           public_ip: "{{ openshift_public_ip | default(None) }}"
           deployment_type: "{{ openshift_deployment_type }}"
+          portal_net: "{{ openshift_portal_net | default(openshift_master_portal_net) | default(None) }}"
```

Comment:
Are we going to merge that shortly, should I simply add things to your maps there if yours merges first?